### PR TITLE
Automatic builds

### PR DESCRIPTION
**Pull request by john-n-smith**

### CODE REVIEW

File: .scrutinizer.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# .scrutinizer.yml
+
+checks:
+    php:
+        custom_coding_standard:
+             git_repository: 'https://github.com/graze/standards'
+             git_version: 'ff7ea57e14cc222ff2c29f36a0bf40e707783bbb'
+             ruleset_path: 'PHP/CodeSniffer/Graze/ruleset.xml'
```
Comment (h-bragg) [inline .scrutinizer.yml]: do we need to use git stuff here if we are pulling in the ruleset.xml file into vendor/graze/standards/...?

Comment (john-n-smith) [inline .scrutinizer.yml]: Unfortunately it's not supported. See my feature request [here](https://github.com/scrutinizer-ci/scrutinizer/issues/380).

Comment (h-bragg) [inline .scrutinizer.yml]: ah, cool. And we have to update the version manually whenever something changes?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	$(DOCKER) rmi ${DOCKER_REPOSITORY}:latest
+
+
+# Testing
+
+test: ## Run the unit and integration testsuites.
+test: lint test-unit
+
+lint: ## Run phpcs against the code.
+	$(DOCKER_RUN) vendor/bin/phpcs -p --standard=vendor/graze/standards/PHP/CodeSniffer/Graze/ruleset.xml --warning-severity=0 src/ tests/
```
Comment (h-bragg) [inline Makefile]: based on your comment in the graze/web PR. I assume we don't need this `--standard` parameter? as it will be read from `phpcs.xml`?

Comment (john-n-smith) [inline Makefile]: Yep, once that is merged I will add to `graze/standards` and update the other projects I look after separately.

File: .scrutinizer.yml (hunk 1 above)
Comment (john-n-smith) [inline .scrutinizer.yml]: Yep, it sucks.

File: Makefile (hunk 2 above)
Comment (h-bragg) [inline Makefile]: Isn't is not required here as you created a `phpcs.xml` file in this repository? so the command can be just `$(DOCKER_RUN) vendor/bin/phpcs -p --warning-severity=0 src/ tests/` ?

Comment (john-n-smith) [inline Makefile]: Yep.

**Comment by h-bragg** [comment]
Looks good to me.

I need to add that config to all of my repositories too

**Comment by h-bragg** [comment]
A few issues on Scrutinizer (Major and Minor).

Also I just install the chrome extension, adds coverage and issue addition/resolution to each commit/pr.

**Comment by john-n-smith** [comment]
@h-bragg, thanks for looking over. I've got a separate PR to address the scrutinizer issues. Would you mind sticking an 'approved' on this?

**Comment by h-bragg** [comment]
There doesn't appear to be any tags, so I have done a :+1:

**Pull request merged**